case_studies/CARD/Bug-Drug Code.R

```diff
@@ -0,0 +1,154 @@
+# config.R
+url <- "https://card.mcmaster.ca/download/0/broadstreet-v3.3.0.tar.bz2"
+destfile <- "broadstreet-v3.3.0.tar.bz2"
+
+# Download the file
+download.file(url, destfile)
+
+#Extract the file
+install.packages("R.utils")
+library(R.utils)
+
+# Decompress the file
+bunzip2("broadstreet-v3.3.0.tar.bz2", destname = "broadstreet-v3.3.0.tar", remove = FALSE)
+file.rename("broadstreet-v3.3.0.tar", "broadstreet-v3.3.0_old.tar")
```

Comment:
This steps are not clear to me.

Reply:
Line 13 decompresses the zipped broadstreet v3.3.0 file using the bunzip function
For line 14, I had to rename the files when trying to debug my code. Since I had to rerun the code multiple times, I didn't want to get confused and so decided to rename the file. I have now update the code to remove the file.rename step